build.gradle

```diff
@@ -285,9 +285,16 @@ project(':iceberg-aliyun') {
     implementation project(':iceberg-common')
 
     compileOnly 'com.aliyun.oss:aliyun-sdk-oss'
-    compileOnly 'javax.xml.bind:jaxb-api'
-    compileOnly 'javax.activation:activation'
-    compileOnly 'org.glassfish.jaxb:jaxb-runtime'
+    if (jdkVersion <= '8') {
+      compileOnly 'javax.xml.bind:jaxb-api'
+      compileOnly 'javax.activation:activation'
+      compileOnly 'org.glassfish.jaxb:jaxb-runtime'
+    } else {
```

Comment:
For JDK8, we don't need to add those to compile classpath because the aliyun document says only greater JDK versions need this.